float64 Mx
float64 My
float64 Fz
